content/releases.md

```diff
@@ -268,6 +274,13 @@ may be completely refactored or have breaking changes in _minor_ releases.
 
 We will try not break the tool in _patch_ releases.
 
+### Daemon Configuration
+
+The daemon's configuration file, commonly located in `/etc/containerd/config.toml`
+is versioned and backwards compatible.  The `version = 2` field in the config
```

Comment:
The `version` field in the config file specifies the config's version. If no version number... Use `version = 2` to enable/access ....